frontend/src/components/dashboard/PremiumOnboarding.tsx

```diff
@@ -384,21 +388,69 @@ const StepThree = () => {
   const { l10n } = useLocalization();
   const isLargeScreen = useMinViewportWidth("md");
 
+  let addonDescription = null;
+  let title = l10n.getString("multi-part-onboarding-reply-headline");
+  if (isLargeScreen && supportsFirefoxExtension()) {
```

Comment:
Should the banners still be visible on mobile? I am still seeing the banners at a device width of < `500px`

Reply:
![image](https://user-images.githubusercontent.com/13066134/160896245-0af4312a-78d9-4686-ae93-9bb258a2720e.png)

Reply:
Oh, that's a good point! This file specifically is for the Premium onboarding flow, but the profile banners did indeed not yet look at screen size to determine which banners to show. Fixed in f6e61f6a.